src/agent/onefuzz-agent/src/worker/tests.rs

```diff
@@ -253,10 +253,16 @@ sys.stderr.write('C' * 65536 + 'D' * 4000)";
     redirected.child.wait().unwrap();
     let captured = redirected.streams.unwrap().join().unwrap();
 
-    let stdout: String = repeat('A').take(96).chain(repeat('B').take(4000)).collect();
+    let stdout: String = repeat('A')
+        .take(MAX_TAIL_LEN - 4000)
+        .chain(repeat('B').take(4000))
```

Comment:
Do these values need to be higher as well? It looks like the code was just reformatted.

Reply:
i made the test dependent on the size of the buffer. The previous version assumed (MAX_TAIL_LEN == 4096)